### PR TITLE
Added the DeleteFile and DeleteDirectory to the FTP client libraries

### PR DESCRIPTION
**Pull request by mortoize**

### CODE REVIEW

File: Ark.Tools.FtpClient.FluentFtp/FluentFtpClientConnection.cs [hunk 1]
```diff
@@ -2,6 +2,9 @@
 // Licensed under the MIT License. See LICENSE file for license information. 
 using Ark.Tools.FtpClient.Core;
 using FluentFTP;
+
+using Microsoft.Identity.Client;
```
Comment (AndreaCuneo) [inline Ark.Tools.FtpClient.FluentFtp/FluentFtpClientConnection.cs:6]: this should not be needed

File: Ark.Tools.FtpClient.SystemNetFtpClient/SystemNetFtpClientConnection.cs [hunk 2]
```diff
@@ -1,5 +1,8 @@
 ﻿// Copyright (c) 2018 Ark S.r.l. All rights reserved.
 // Licensed under the MIT License. See LICENSE file for license information. 
+
+using Microsoft.Identity.Client;
```
Comment (AndreaCuneo) [inline Ark.Tools.FtpClient.SystemNetFtpClient/SystemNetFtpClientConnection.cs:4]: ditto

File: global.json [hunk 3]
```diff
@@ -1,6 +1,6 @@
 {
   "sdk": {
-    "version": "6.0.403",
```
Comment (AndreaCuneo) [inline global.json:3]: currently we don't aim to target net7 as is not LTS.
please revert to .net 6.0

File: Directory.Build.props [hunk 4]
```diff
@@ -1,7 +1,7 @@
 <Project>
   
   <PropertyGroup>
-    <TargetFrameworks>net6.0;netstandard2.1;netstandard2.0;net472</TargetFrameworks>
+    <TargetFrameworks>net7.0;net6.0;netstandard2.1;netstandard2.0;net472</TargetFrameworks>
```
Comment (AndreaCuneo) [inline Directory.Build.props:4]: currently we don't aim to target net7 as is not LTS.
please revert this.

**Review by AndreaCuneo — CHANGES_REQUESTED** [review]
@mortoize thank you for your feature contribution.

I've flagged a few changes, but overall I'm fine with accepting this and release once ready.
Cheers

**Comment by mortoize** [comment]
> I've flagged a few changes, but overall I'm fine with accepting this and release once ready. Cheers

@AndreaCuneo Thank you for your input. The changes you've requested are done. Hope, everything goes well.

**Review by AndreaCuneo — APPROVED** [review]

**Comment by AndreaCuneo** [comment]
included in v4.7.0

**Pull request merged**